gems/quilt_rails/README.md

```diff
@@ -829,10 +834,28 @@ The `Performance::Reportable` mixin uses [https://github.com/Shopify/statsd-inst
 
 ### Generators
 
-#### `quilt:install`
+#### `quilt_rails:install`
+
+Run both generator below.
```

Comment:
IMO, this section is structured a bit confusing for newcomers. I would split it up in 2 sections.  

* One section for the "umbrella" generators. (ie. those that have "Run ... below") . Also, I have a feeling 90% of folks will want to run the "umbrella" generators so providing a code snippet of how it's run might be useful. (eg. `bundle exec` or `rails generate` or w/e thing you need to run). Maybe also explicitly mention what it runs instead of using the term "below" if we decide to split it up into two sections. 
* One section for details explanations of what the individual commands do.

Reply:
> Also, I have a feeling 90% of folks will want to run the "umbrella" generators so providing a code snippet of how it's run might be useful. (eg. bundle exec or rails generate or w/e thing you need to run)

This is the job of the `Generate app boilerplate` section above. This part of the doc is mostly to explain the individual commands. I will re-work it thou  :)